File: app/src/main/java/com/chaity/android/easy/move/dagger/builders/ActivityBuilder.java
package com.chaity.android.easy.move.dagger.builders;



import com.chaity.android.easy.move.ui.delivery.DeliveriesActivity;
import com.chaity.android.easy.move.ui.delivery.DeliveryActivityModule;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;


@Module
public abstract class ActivityBuilder {



    @ContributesAndroidInjector(modules = DeliveryActivityModule.class)
    abstract DeliveriesActivity contributeDeliveriesActivity();




}

File: app/src/main/java/com/chaity/android/easy/move/dagger/builders/ActivityBuilder.kt
package com.chaity.android.easy.move.dagger.builders


import com.chaity.android.easy.move.ui.delivery.DeliveryActivity
import com.chaity.android.easy.move.dagger.modules.activityModules.DeliveryActivityModule

import dagger.Module
import dagger.android.ContributesAndroidInjector


@Module
abstract class ActivityBuilder {

    @ContributesAndroidInjector(modules = [DeliveryActivityModule::class])
    internal abstract fun contributeDeliveriesActivity(): DeliveryActivity


}
